src/install/dotnet/intro.mdx

```diff
@@ -12,3 +12,5 @@ New Relic supports the most popular .NET application implementations. Once insta
 />
 
 ## Installation options [#install]
+
+The New Relic .NET agent has installation options for Linux, Windows, AWS elastic beanstalk, WCF, Nuget, and Azure cloud, web, and service fabric. Select your installation path using the form below for installation instructions based on your environment!
```

Comment:
The New Relic .NET agent has installation options for Linux, Windows, AWS elastic beanstalk, WCF, Nuget, Azure cloud, web, and service fabric.